CONTRIBUTING.md

```diff
@@ -234,12 +252,12 @@ In each file, delete section headers corresponding to sections that are empty.
 
 <#includes for header files belonging to our project>
 
-///////////////////////   Macros and Type Definitions   ////////////////////////
 <preprocessor definitions>
 
 <definitions for public types>
 
-///////////////////////           Procedures             ///////////////////////
+<global variable declarations>
+
 <public procedure prototypes>
 
 #endif // <file name in all caps, with "." replaced with "_">
```

Comment:
Looks like there are stray `//` here?

Reply:
I do not see them. After the `#endif`, the file name should appear in a comment.